modules/dice_loss_metric_notes.ipynb

```diff
@@ -1,5 +1,22 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6459ddb8",
+   "metadata": {},
+   "source": [
+    "Copyright (c) MONAI Consortium  \n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\");  \n",
+    "you may not use this file except in compliance with the License.  \n",
+    "You may obtain a copy of the License at  \n",
+    "http://www.apache.org/licenses/LICENSE-2.0  \n",
```

Comment:
This should be 4 spaces before the first letter. 
"    http://www.apache.org/licenses/LICENSE-2.0  \n",

Reply:
Hi @mingxin-zheng, I just noticed that your template has the same same format with @Nic-Ma.
https://github.com/Project-MONAI/tutorials/blob/main/.github/contributing_templates/notebook/example_feature.ipynb
Do I miss anything?
Thanks!

Reply:
Updated in the latest commit.

Thanks.